assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -160,8 +160,21 @@ cmdline_additionalArg=+{{.AdditionalArgs}}
 
 {{if .PerPodPowerManagement}}
 cmdline_pstate=+intel_pstate=passive
+{{else if .HardwareTuning}}
+cmdline_pstate=+intel_pstate=active
 {{else if .RealTimeHint}}
 cmdline_pstate=+intel_pstate=disable
 {{end}}
 
 [rtentsk]
+
+{{ if .HardwareTuning }}
+[sysfs]
+# sets provided frequencies to isolated and reserved cpus
+{{ range .IsolatedCpuList }}
```

Comment:
It would be nice to add a unit test for the ranges testing different formats (comma separated lists , range lists, combined)

Reply:
Hi @yanirq, i have replied to your comment in the following link:
https://github.com/openshift/cluster-node-tuning-operator/pull/908/files#r1477824265